packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -26,17 +26,13 @@ import {
 } from '../constants';
 import { SObjectDescribe } from '../describe';
 import { nls } from '../messages';
-import {
-  ChildRelationship,
-  Field,
-  SObject,
-  SObjectCategory,
-  SObjectRefreshSource
-} from '../types';
+import { SObject, SObjectCategory, SObjectRefreshSource } from '../types';
 import { ConfigUtil } from './configUtil';
+import { DeclarationGenerator, MODIFIER } from './declarationGenerator';
+import { TypingGenerator } from './typingGenerator';
 
+const TYPING_PATH = ['typings', 'lwc', 'sobjects2'];
```

Comment:
I haven't figured out how to do the transition yet. ALS will collide with this typing generation until it is disabled.

Reply:
We can have a "local" variable/setting to gate this being enabled but still allow us to write tests, merge it, etc.